content/post/language_migrations.md

```diff
@@ -512,6 +512,158 @@ The thickness of each band corresponds to the value in the dominant eigenvector.
 
 * Regarding **Go**, the popularity stays relatively low. However, the dynamics is clearly positive.
 
+## Edit
+
+In this extra part, we want to answer to people's relevant concerns that our global quantization of contributions would benefit to verbose languages like **Java** over concise ones like **Haskell**. Actually, when applying the quantization trick to each specific language, you can see, on the table below, small moves expected as **Java** that loses some popularity. However, the ranking stays nearly the same ; only **Ruby** and **C++**  are switching position but percentages are really close from each other in this part of the table. Besides the final stack area plot looks exactly the same. For those reasons, we decide to stick to the global quantization.
```

Comment:
I read some concerns about the language verbosity bias after publishing the post. They are fair: the global quantization scheme may give an advantage to verbose languages like **Java** in difference to condensed ones like **Haskell**. I quantized each of the languages independently and re-run the rest of the analysis. As you can see in the table below, nothing really changed; **Ruby** and **C++** exchanged the position, but their ranks are really close to each other. The final history plot looks **exactly** the same.